pkg/apis/options/options.go

```diff
@@ -216,6 +216,8 @@ func NewFlagSet() *pflag.FlagSet {
 	flagSet.Bool("redis-use-cluster", false, "Connect to redis cluster. Must set --redis-cluster-connection-urls to use this feature")
 	flagSet.StringSlice("redis-cluster-connection-urls", []string{}, "List of Redis cluster connection URLs (eg redis://HOST[:PORT]). Used in conjunction with --redis-use-cluster")
 
+	flagSet.String("dynamodb-table-name", "", "DynamoDb table to store session state in")
```

Comment:
NIT: I think `dynamodb-table` is clear enough that we are looking for the name of the table while keeping the flag more consise.